winsdk/storage.go

```diff
@@ -244,7 +244,8 @@ func MultiUpload(_allocationID, _workdir, _jsonMultiUploadOptions *C.char, statu
 	if err != nil {
 		return WithJSON(nil, err)
 	}
-	err = a.StartMultiUpload(workdir, filePaths, fileNames, thumbnailPaths, encrypts, remotePaths, false, &StatusCallbackWrapped{Callback: statusCb})
+
+	err = a.StartMultiUpload(workdir, filePaths, fileNames, thumbnailPaths, encrypts, remotePaths, false, &StatusCallbackWrapped{Callback: nil})
 	if err != nil {
```

Comment:
`upload progress` should not be hidden. it is better to send it to client. so we can build more friendly UI/UX .

Reply:
This PR is still in WIP. I have temporary removed this to check if it was the reason for the error.

Reply:
added callback function